platform/windows/nuget.cmake

```diff
@@ -21,13 +33,14 @@ set(CPACK_NUGET_PACKAGE_DESCRIPTION "
   `--params`. For example: `--params='/InstallService'`.
   "
 )
+set(OSQUERY_RAW "https://raw.githubusercontent.com/osquery/osquery/")
 set(OSQUERY_REPO "https://github.com/osquery/osquery/")
```

Comment:
Is this no longer used?

Reply:
Still used for the release notes reference.